File: Ecuui/python/run/python_code_run.py
import os
from delphivcl import *

class Form6(Form):

    def __init__(self, owner):
        self.Memo1 = None
        self.Memo2 = None
        self.Panel1 = None
        self.Button1 = None
        self.Button2 = None
        self.Button3 = None
        self.OpenDialog1 = None
        self.SaveDialog1 = None
        self.LoadProps(os.path.join(os.path.dirname(os.path.abspath(__file__)), "python_code_run.pydfm"))

    def Button1Click(self, Sender):
        pass

    def Button2Click(self, Sender):
        pass

    def Button3Click(self, Sender):
        pass

File: Ecuui/python/python_code_run.pas
unit python_code_run;

interface

uses
  Winapi.Windows, Winapi.Messages, System.SysUtils, System.Variants, System.Classes, Vcl.Graphics,
  Vcl.Controls, Vcl.Forms, Vcl.Dialogs, PythonEngine, Vcl.PythonGUIInputOutput,
  Vcl.StdCtrls, Vcl.ExtCtrls;

type
  TForm6 = class(TForm)
    Memo1: TMemo;
    Memo2: TMemo;
    Panel1: TPanel;
    Button1: TButton;
    Button2: TButton;
    Button3: TButton;
    OpenDialog1: TOpenDialog;
    SaveDialog1: TSaveDialog;
    Label1: TLabel;
    Panel2: TPanel;
    ButGenLed: TButton;
    ButShowmap: TButton;
    Button6: TButton;
    Label2: TLabel;
    Label3: TLabel;
    procedure Button1Click(Sender: TObject);
    procedure Button2Click(Sender: TObject);
    procedure Button3Click(Sender: TObject);
  private
    { Private declarations }
  public
    { Public declarations }
  end;

var
  Form6: TForm6;

implementation

{$R *.dfm}

procedure TForm6.Button1Click(Sender: TObject);
var
  Result : PPyObject;
begin
  
end;


procedure TForm6.Button2Click(Sender: TObject);
begin
  with OpenDialog1 do
    begin
      if Execute then
        Memo1.Lines.LoadFromFile( FileName );
    end;
end;

procedure TForm6.Button3Click(Sender: TObject);
begin
  with SaveDialog1 do
    begin
      if Execute then
        Memo1.Lines.SaveToFile( FileName );
    end;
end;

end.
